.env.mainnet

```diff
@@ -1,5 +1,5 @@
-TIME_LOCK_DELAY = 7
-TIME_LOCK_MIN_DELAY = 4
+git cTIME_LOCK_DELAY = 25200
```

Comment:
`git c` by accident?
To these seconds amounts, add comments specifying how that translates to minutes/hours/days.